src/components/AddressSearch/index.js

```diff
@@ -254,6 +255,30 @@ function AddressSearch(props) {
             values.state = stateFallback;
         }
 
+        // Some edge-case addresses may lack both street_number and route in the API response, resulting in an empty "values.street"
+        // We are setting up a fallback to ensure "values.street" is populated with a relevant value
+        if (!values.street) {
+            const {adr_address} = details;
+            const streetAddressRegex = /(<span\s+class\s*=\s*"street-address"[^>]*>[^<]*<\/span>)/;
+            const streetAddressHtml = adr_address.match(streetAddressRegex);
+            if (streetAddressHtml[1]) {
+                const parser = new ExpensiMark();
+                const streetFallback = parser.htmlToText(streetAddressHtml[1]);
+                values.street = streetFallback;
+            } else {
```

Comment:
I think we can just leave this part out for this PR, it feels a bit too hacky to me.

Reply:
Do you mean not use adr_address or the parser itself?

Reply:
Ooh sorry the highlight didn't work, I meant the fallback to the fallback isn't necessary.

Reply:
Ahhhh got you, Okay I'll just focus on ad_address then